k8s/cms-transcode-worker/deployment.yaml

```
@@ -0,0 +1,25 @@
+      app: cms-transcode-worker
+  template:
+    metadata:
+      labels:
+        app: cms-transcode-worker
+    spec:
+      containers:
+      - name: cms-transcode-worker
+        image: ghcr.io/callein/cms-transcode-worker:latest
+        command: [ "celery", "-A", "src.worker.celery_worker.celery_app", "worker", "--loglevel=info", "--concurrency=1", "-Q", "celery.transcode" ]
+        envFrom:
+          - secretRef:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)